server/lib/celluloid_tracer.rb

```diff
@@ -0,0 +1,17 @@
+module CelluloidCallTracer
+  def method_missing(method, *args, &block)
```

Comment:
This is just overriding the existing `Celluloid::Proxy::*#method_missing` to log calls